services/policies/pkg/config/config.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import (
+	"context"
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+	"time"
+)
+
+// Config combines all available configuration parts.
+type Config struct {
+	Commons           *shared.Commons       `yaml:"-"` // don't use this directly as configuration for a service
+	GRPC              GRPC                  `yaml:"grpc"`
+	Service           Service               `yaml:"-"`
+	TokenManager      *TokenManager         `yaml:"token_manager"`
+	Events            Events                `yaml:"events"`
+	Reva              *shared.Reva          `yaml:"reva"`
+	GRPCClientTLS     *shared.GRPCClientTLS `yaml:"grpc_client_tls"`
+	MachineAuthAPIKey string                `yaml:"machine_auth_api_key" env:"OCIS_MACHINE_AUTH_API_KEY;POLICIES_MACHINE_AUTH_API_KEY" desc:"Machine auth API key used to validate internal requests necessary for the access to resources from other services."`
+	Context           context.Context       `yaml:"-"`
+	Log               *Log                  `yaml:"log"`
+	Engine            Engine                `yaml:"engines"`
+	Postprocessing    Postprocessing        `yaml:"postprocessing"`
+}
+
+// Service defines the available service configuration.
+type Service struct {
+	Name string `yaml:"-"`
+}
+
+// GRPC defines the available grpc configuration.
+type GRPC struct {
+	Addr      string                 `ocisConfig:"addr" env:"POLICIES_GRPC_ADDR" desc:"The bind address of the GRPC service."`
+	Namespace string                 `ocisConfig:"-" yaml:"-"`
+	TLS       *shared.GRPCServiceTLS `yaml:"tls"`
+}
+
+// TokenManager is the config for using the reva token manager
+type TokenManager struct {
+	JWTSecret string `yaml:"jwt_secret" env:"OCIS_JWT_SECRET;POLICIES_JWT_SECRET" desc:"The secret to mint and validate jwt tokens."`
+}
+
+// Engine configures the policy engine.
+type Engine struct {
+	Timeout  time.Duration `yaml:"timeout" env:"POLICIES_ENGINE_TIMEOUT" desc:"Sets the timeout."`
```

Comment:
I would like to do a suggestable commit but I need to clarify first...

* Do I guess that this is the the timeout how long the rego evaluations are allowed to take?
* Do we have an "unlimited" value?
* Do we have a default time unit like seconds?
* Can we use abbreviations like [ms], [s] or similar?

Reply:
> * Do I guess that this is the the timeout how long the rego evaluations are allowed to take?
yes

> * Do we have an "unlimited" value?
no, the policy should evaluate quick, if the evaluation takes too long (it should take more than some ms) something is fishy.

> * Do we have a default value?
yes, 10 seconds which is way too much

> * Do we have a default time unit like seconds?
yes, it uses time.Duration, values like 1s are fine.

> * Can we use abbreviations like [ms], [s] or similar?
yes, see above

Reply:
What is the default response is an evaluation is timed out? block or pass?

Reply:
Post clarification, I added a suggestable commit.